setup.py

```diff
@@ -50,7 +61,7 @@ def _plugin_version() -> str:
     install_requires=[
         "dbt-common>=0.1.0a1,<2.0",
         "dbt-adapters>=0.1.0a1,<2.0",
-        f"dbt-postgres~={_plugin_version()}",
+        f"dbt-postgres~={_plugin_version_trim()}",
```

Comment:
@mikealfare can this be done without `~=`?

Reply:
I would need to look at it, but I would think so. We would need to update the function to return a tuple, something along the lines of `"dbt-postgres>={},<{}".format(_plugin_version())` where `_plugin_version() == (1.8.0b1, 1.9)`.